### ISSUE TITLE
Plugin not responsive to QGIS Darkmode. 

### ISSUE THREAD

**Issue by BallJonathan** [koordinates/koordinates-qgis-plugin#166]
**Describe the bug**
The plugin is not readable when QGIS dark mode is active. Please block dark mode setting from affecting the plugin design.  
![image](https://user-images.githubusercontent.com/113562913/203433767-0a46dcd0-a46c-4530-a585-422a9e366fbd.png)

**Comment by nyalldawson** [comment]
@BallJonathan I don't believe it's possible to selectively disable dark theme on just one part of qgjs. Would it be possible to instead get mockups for a dark theme friendly colouring for the design? We could respond gracefully to dark mode then.

**Comment by BallJonathan** [comment]
Could we hard-code the styles? I see in the screenshot above, the cards seem to be able to preserve their intended appearance. How is that done?

**Comment by nyalldawson** [comment]
> Could we hard-code the styles? I see in the screenshot above, the cards seem to be able to preserve their intended appearance. How is that done?

There's two options I can see for solving this:

1. Remove the part where we respect theme colors in the white cards items, which is causing the white-on-white text, and instead hardcode black for the invisible text elements. That would solve the readability of the cards, but we'd still have issues like the almost-invisible black on dark grey tab and button icons.
2. Properly respond to dark themes -- this is obviously a "better" solution (since the dock won't be a glaring panel on dark themes!), but is a little more involved. To solve it we'd need:

- variants on the icons with "light" coloring
-  removal of all hardcoded colors and ensure we always take colors from the application "palette" instead
- (somehow nicely handle company logos, which might result in black-on-black again! :scream:  )

**Comment by BallJonathan** [comment]
So just to confirm, there is no way to block all QGIS or OS theme settings
from affecting the colours of the plugin?

Jonathan Ball
koordinates.com / @koordinates <http://twitter.com/koordinates>
